ncbi-amrfinderplus/3.12.8-2024-07-22.1/README.md

```diff
@@ -0,0 +1,68 @@
+# NCBI AMRFinderPlus docker image
+
+Main tool : [NCBI AMRFinderPlus](https://github.com/ncbi/amr)
+
+Additional tools:
+
+- hmmer v3.3.2
+- ncbi-blast+ v2.16.0
+
+## Database information
+
+The database included at time of docker image build is **`2024-05-02.2`**. More information can be found in the [changelog.txt on NCBI's FTP](https://ftp.ncbi.nlm.nih.gov/pathogen/Antimicrobial_resistance/AMRFinderPlus/database/3.12/2024-05-02.2/changelog.txt).
```

Comment:
Can you please update this line with the correct db version and changelog link? 

https://ftp.ncbi.nlm.nih.gov/pathogen/Antimicrobial_resistance/AMRFinderPlus/database/3.12/2024-07-22.1/changelog.txt

Reply:
Done!